core/snapshots/testsuite/testsuite.go

```diff
@@ -1052,6 +1052,29 @@ func checkWalk(ctx context.Context, t *testing.T, snapshotter snapshots.Snapshot
 		t.Fatal(err)
 	}
 
+	// Base parent with quotasize=1G
+	if _, err := snapshotter.Prepare(ctx, "p-qs-tmp", "", opt); err != nil {
+		t.Fatal(err)
+	}
+	if err := snapshotter.Commit(ctx, "p-qs", "p-qs-tmp", snapshots.WithQuotaSize("1G")); err != nil {
+		t.Fatal(err)
+	}
+
+	// active with quotasize=2G
+	if _, err := snapshotter.Prepare(ctx, "a-qs", "p-qs", snapshots.WithQuotaSize("2G")); err != nil {
+		t.Fatal(err)
+	}
+
+	// view with quotasize=3G
+	if _, err := snapshotter.View(ctx, "v-qs", "p-qs", snapshots.WithQuotaSize("3G")); err != nil {
+		t.Fatal(err)
+	}
+
+	// no parent active with quotasize=4G
+	if _, err := snapshotter.Prepare(ctx, "a-np-qs", "", snapshots.WithQuotaSize("4G")); err != nil {
+		t.Fatal(err)
+	}
+
```

Comment:
Test should write something beyond the quota size and check how it fails

Reply:
`testsuite.go` seems to be used for integration test, while now `snapshotter_test.go` is used for unit test